pom.xml

```diff
@@ -66,10 +76,124 @@
         <dependency>
             <groupId>com.massivecraft.factions</groupId>
             <artifactId>Factions</artifactId>
-            <version>2.13.7</version>
+            <version>2.14.0</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <dependency>
+            <groupId>com.massivecraft.massivecore</groupId>
+            <artifactId>MassiveCore</artifactId>
+            <version>2.14.0</version>
```

Comment:
I found the repos for them, so I added them...

Reply:
https://repo.dustplanet.de/artifactory/ext-release-local/ - not working (temporary ?)

towny - no repository ?

Reply:
you need to add towny to your local maven repo yourself.
Just find out where your local maven repo is, download towny and drag the file to <MAVEN_REPO>/towny/towny/0.93/towny-0.93.jar

Reply:
Okay, but what is with "dustplanet" - massivecore ?

Reply:
The repo appearently got shut down a few days ago... I will see, if there is another one. If not, I will add my repo

Reply:
Pull and try again please. The repo now works